spec/factories/comment.rb

```diff
@@ -4,11 +4,11 @@
     text { FFaker::CheesyLingo.paragraph }
 
     factory :team_comment do
-      team
+      commentable { create :team }
```

Comment:
The `association` syntax used to be broken for polymorphic, but afaik on the current FactoryGirl version you can now use:

```ruby
association :commentable, factory: :team
````

But comes down to personal preference in the end @carpodaster 😉